styles/color-mixins.less

```diff
@@ -145,11 +145,11 @@
 }
 .sand-button-selected-bg-color () {
 	background-color: @sand-button-bg-color;
-	filter+_: saturate(1.5) brightness(0.5); // Simulates #3e454d, the spec color
+	filter+_: @sand-button-selected-filter // Simulates #3e454d, the spec color
```

Comment:
Please remove the comment

Reply:
Please do the same in https://github.com/enactjs/sandstone/pull/984.

Reply:
✅